### PR TITLE
print stylesheet

### PR DESCRIPTION
**Pull request by kdoh**
addresses #1472 
- [x] rip off chrome when printing 
  screen shot 2016-10-25 at 10 30 01 am
- [x] make dashcards more predictable when printing (@tlrobinson I could use some guidance on how we think we should go about this).

⬇️  **Please click the 👍 reaction instead of leaving a `+1` or 👍  comment**

### CODE REVIEW

**Comment by tlrobinson** [comment]
@kdoh what else needs to be done for this? I think I've got it working pretty well for standard 8.5"x11" letter sized paper in portrait or landscape orientation. Not sure how it will work on other paper though, or whether we can make it work for all paper sizes, but this might be a good enough start.

We might also want to show the dashboard title. And maybe a timestamp?

**Comment by kdoh** [comment]
Looks pretty good to me as far as the 8.5 x 11 preview goes. I'll try a couple test prints just to verify here in a few.

I can see your point about the dash title so we may want to not hide that. I think as far as timestamps go I'm thinking we should probably leave that up to the native print dialogues.

**Comment by kdoh** [comment]
Need to make a couple CSS tweaks to remove some distracting card elements (like drop shadows, etc) and then I think we should be good to go. As far as titles go I think we should be using the page title the print dialogue can add rather than taking up non header / footer space on the print document itself.

File: frontend/src/metabase/css/dashboard.css [hunk 1]
```diff
@@ -477,17 +477,27 @@
...
     }
+    .DashCard .Card {
+      border: none !important;
+      box-shadow: none !important;
+    }
+    /* improve label contrast */
+    .dc-chart .axis .tick text,
+    .dc-chart .x-axis-label,
+    .dc-chart .y-axis-label {
+      fill: #222222 !important;
```
Comment (tlrobinson) [inline frontend/src/metabase/css/dashboard.css]: `!important`

:(

Comment (kdoh) [inline frontend/src/metabase/css/dashboard.css]: Hrm. Yeah on second thought it looks like we don't even need it. Good catch.

**Comment by tlrobinson** [comment]
It looks a bit weird without borders IMHO screen shot 2016-11-07 at 3 38 32 pm

**Comment by kdoh** [comment]
Re-added the borders. This should be good to go now.

**Comment by ithasyou** [comment]
Not working anymore, lastest version of metabase
image

Thanks for your support!

**Comment by franziz** [comment]
+1 for not working

**Comment by mickdelaney** [comment]
+1 for not working. 

Anyone got any work arounds ?

**Comment by maajesus** [comment]
Hi, 

For me on v0.29.3 is working but when I print the series color in the PDF doesnt appear and becames hard to understand if a serie corresponds to wich line. The same for pie charts. Any workaround?

Thanks in advance 

image

image

**Pull request merged**